examples/00-systemcoupling/oscillating_plate.py

```diff
@@ -45,19 +45,19 @@
    :align: center
 
 There is no friction between the plate and the side of the cavity. An
-initial pressure of 100 Pa is applied to one side of the thin plate
-for 0.5 seconds to distort it. Once this pressure is released, the plate
+initial constant force in x-direction is applied to one side of the thin plate
+for the first 0.5 seconds to distort it. Once this pressure is released, the plate
 oscillates back and forth to regain its equilibrium, and the
 surrounding air damps this oscillation. The plate and surrounding
 air are simulated for a few oscillations to allow an examination of the
 motion of the plate as it is damped.
 
 """
 # %%
-# Set up example
-# --------------
+# Import modules, download files, launch products
+# -------------------------------------
```

Comment:
Just to note that Sphinx is very fussy about underlines. These need to be exactly the same length as the text on the line above.